.github/workflows/frontend-test.yml

```diff
@@ -11,6 +11,9 @@ on:
   pull_request:
     branches:
       - 'main'
+env:
+  BRANCH_NAME: ${{ github.head_ref || github.ref_name }}
```

Comment:
What's this getting used for?

Reply:
`api/scripts/run` runs the dockerised api using the current branch name as a tag